examples/executable.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace executable;
+
+use Castor\Attribute\AsTask;
+
+use function ls\ls;
+
+#[AsTask(name: 'ls', description: 'Run a sub-process and display information about it')]
+function ls_task(): void
+{
+    ls(__DIR__ . '/executable')->run();
```

Comment:
I would use a "custom" method here, to better show how it works

```suggestion
    ls(__DIR__ . '/executable')->all()->run();
```

Reply:
Yeah but all give info about user / other stuff so tests won't be the same given the env

Reply:
all() should return always the same things 🤔

Reply:
My bad i was thinking about `long()`